ONSdigital/ras-frontstage
# Change planet font classes to design system versions

# Motivation and Context
The pattern library use of planets to determine text size is deprecated in favour of utility font size classes.  Updates the version of the pattern library to the most recent, updates the font classes and fixes some javascript that broke as a result.

# What has changed
- Updated the version of the pattern library used to 2.0.0.
- Updated the planet classes to utility font sizes.  
- Fixed the guidance section that broke as part of the update.

# How to test?
Look at as many pages as possible and ensure it looks the same as the current ras-frontstage.  Also ensure no functionality is broken as a result.

# Links
https://trello.com/c/C74y7xaL/1074-change-frontstage-to-use-the-utility-classes-in-the-html-instead-of-the-planets-8
